Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```
@@ -3,76 +3,80 @@
 span.bocEnumValue,
 div.bocEnumValue.readOnly
 {
-  width: 15em;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  overflow: hidden;
+  display: inline-flex;
+  width: var(--remotion-themed-control-width);
+  min-height: var(--remotion-themed-min-height);
 }
 
-.bocEnumValue.readOnly
+.bocEnumValue td
 {
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
+  white-space: nowrap;
 }
 
-.bocEnumValue.disabled
+.bocEnumValue td label
 {
+  padding: var(--remotion-themed-padding);
 }
 
-span.bocEnumValue select
+div.bocEnumValue td,
+div.bocEnumValue td span
 {
-  width: 100%;
-  height: 1.68em;
+  display: inline-flex;
+  align-items: center;
+  box-sizing: border-box;
 }
 
-span.bocEnumValue select[size]
+div.bocEnumValue input
 {
-  height: auto;
+  /* Has to be set explicitely for input. */
+  margin: 0;
 }
 
-span.bocEnumValue select[size="1"]
+div.bocEnumValue input:focus
 {
-  height: 1.68em;
+  /* In order to have an outline for the radio buttons as well. */
+  outline: auto;
 }
 
-.bocEnumValue td
+div.bocEnumValue input:not([disabled]):hover
 {
-  white-space: nowrap;
+  cursor: pointer;
 }
 
-.bocEnumValue td label
+span.bocEnumValue select:not([disabled]):hover,
+span.bocEnumValue:not(.disabled) option:hover
```

I think this is now redundant? or rather, should be 
```suggestion
.bocEnumValue select:not([disabled]) option:hover
```

also, please slit this up into separate blocks.